src/Plugin/Validation/Constraint/CropTypeAspectRatioValidationConstraint.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\crop\Plugin\Validation\Constraint\CropTypeAspectRatioValidationConstraint.
+ */
+
+namespace Drupal\crop\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Supports validating crop type aspect ratio.
+ *
+ * @Constraint(
+ *   id = "CropTypeAspectRatioValidation",
+ *   label = @Translation("Crop Type aspect ratio", context = "Validation")
+ * )
+ */
+class CropTypeAspectRatioValidationConstraint extends Constraint {
+
+  /**
+   * The default violation message.
+   *
+   * @var string
+   */
+  public $message = 'Invalid format of aspect ratio. Enter a ratio in format H:W.';
```

Comment:
Maybe something like:

"Invalid aspect ratio format. Should be defined in H:W form."